### DISCUSSION TITLE
SingleFile disapearred from Safari extension

### DISCUSSION THREAD

**Discussion by asanares** [gildas-lormeau/SingleFile#1063] [General]
**Describe the bug**
Installed *SingleFile for Safari* some weeks ago and it worked just fine.
Today I couln't find anymore the button of *SingleFile* in the *Safari* menue.
Also is it missing in the *Safari* extension pane. 

**To Reproduce**
1. Uninstalled the *SingleFile for Safari* app from program folder with *AppCleaner*.
2. Reinstalled from App Store, restarted *Safari*: still *SingleFile* doesn't show up in the *Safari* extension pane.
3. Even the *SingleFile for Safari* is installed on my system it is still not available in *Safari*.

**Environment**
- macOS 12.6.1 / M1 Pro
- Safari 16.1
- SingleFile for Safari 1.0.4

**Comment by gildas-lormeau** [comment]
I tried to reproduce your issue without using AppCleaner and was not able to reproduce it. Actually, I do not wish to take the risk of doing this test with AppCleaner and corrupting something because I use this machine to develop SingleFile on Mac. Also, I'm not experienced enough with the OS to take this risk.

Did you try this procedure? https://discussions.apple.com/thread/253409221

**Comment by asanares** [comment]
Thanks for your fast answer. The linked procedure solved my problem.

AppCleaner is commonly used to uninstall programs and connected files (cache, plist etc.). You can review and select the files which will be deleted before execution. Deleting this files often helps if programs have problems starting.

**Comment by gildas-lormeau** [comment]
Thank you for the feedback. I'm glad to hear the issue is fixed. 

I will wait a little bit before installing AppCleaner. For now I have a lot of free space :).
